README.md

```diff
@@ -261,11 +246,13 @@ await pc.describeIndex('serverless-index');
 //    dimension: 1536,
 //    metric: 'cosine',
 //    host: 'serverless-index-4zo0ijk.svc.us-west2-aws.pinecone.io',
+//    deletionProtection: 'disabled',
 //    spec: {
+//       pod: undefined,
```

Comment:
Same as above, I think it's preferable to just leave it off completely since it's a serverless index.